### PR TITLE
Multilang: kor translation update

### PR DESCRIPTION
**Pull request by crwusiz**

### CODE REVIEW

File: selfdrive/ui/translations/main_ko.ts [hunk 1]
```diff
@@ -1004,19 +1004,19 @@ location set</source>
...
+        <translation>    openpilot은 기본적으로 &lt;b&gt;안정적 모드&lt;/b&gt;로 주행합니다.    실험적 모드는 안정적 모드에 준비되지 않은 &lt;b&gt;알파 수준 기능&lt;/b&gt;을 활성화 합니다.     실험 모드의 특징은 아래에 나열되어 있습니다    &lt;br&gt;     &lt;h4&gt;🌮 E2E 롱컨트롤 🌮&lt;/h4&gt;     주행모델이 가속과 감속을 제어하도록 합니다. openpilot은 신호등과 정지표지판을 보고 멈추는 것을 포함하여 운전자가 생각하는것처럼 주행합니다. 주행 모델이 주행할 속도를 결정하므로 설정된 속도는 상한선으로만 작용합니다.</translation>
     </message>
     <message>
         <source>openpilot defaults to the car&apos;s built-in ACC instead of openpilot&apos;s longitudinal control on this car. Enable this to switch to openpilot longitudinal control.</source>
         <translation>openpilot은 차량의 내장 ACC로 기본 설정됩니다. 롱컨트롤으로 전환하려면 이 옵션을 활성화하세요.</translation>
     </message>
     <message>
         <source>WARNING: openpilot longitudinal control is experimental for this car and will disable Automatic Emergency Braking (AEB).</source>
-        <translation type="unfinished"></translation>
+        <translation>경고: openpilot 롱컨트롤은 실험적인 기능으로 차량의 AEB를 비활성화합니다.</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_ko.ts:1019]: Do people know what AEB is? Can we spell it out?

Comment (crwusiz) [reply selfdrive/ui/translations/main_ko.ts:1019]: ok.. this fix

**Comment by sshane** [comment]
Thanks for the PR!

**Comment by sshane** [comment]
@crwusiz do you mind opening another PR to translate the updated strings? I just made it so when we alter a string you won't have to re-translate the whole paragraph again!

**Comment by crwusiz** [comment]
> @crwusiz do you mind opening another PR to translate the updated strings? I just made it so when we alter a string you won't have to re-translate the whole paragraph again!

ok..!!

**Pull request merged**